src/main/java/com/example/javamavenjunithelloworld/Hello.java

```diff
@@ -26,6 +26,24 @@ public void setTimes(int times) {
                     MAXIMUM_AMOUNT_OF_TIMES
             ));
         }
+		int i=0;int j=0;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Declare "j" on a separate line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1659)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "j" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)